app/components/Stock/store/actions/chart.actions.js

```diff
@@ -0,0 +1,56 @@
+import Api from '../../../../services/api';
+import { showMessage } from '../../../../store/actions';
+export const GET_CHART = '[STOCK] GET CHART';
+export const GET_CHART_SUCCESS = '[STOCK] GET CHART SUCCESS';
+export const GET_CHART_ERROR = '[STOCK] GET CHART ERROR';
+export const TOGGLE_RANGE = '[STOCK] TOGGLE RANGE';
+
+export const RANGE_1DAY = {
```

Comment:
these objects should be moved to a dedicated file and imported

Reply:
Yeah, i need to figure out a better way to deal with those `RANGE` constants.  I'm trying to avoid using a dedicated global Constants file for component specific constants, as it's a DDD anti-pattern.  So i'll mark this one as a todo on the roadmap.